pkg/properties/properties.go

```diff
@@ -45,42 +51,14 @@ func ConfigSliceToMap(configs []string) (map[string]string, error) {
 	return m, nil
 }
 
-func parseLines(content string) []string {
-	var lines []string
-
-	// Support multiline properties
-	content = strings.ReplaceAll(content, "\\\n", "")
-
-	for _, line := range strings.Split(content, "\n") {
-		line = strings.TrimSpace(line)
-		if line != "" && line[0] != '#' {
-			lines = append(lines, line)
-		}
-	}
-
-	return lines
-}
-
 // ConfigFlagToMap reads key=values pairs from the --config flag and supports configuration values containing commas.
 func ConfigFlagToMap(configs []string) (map[string]string, error) {
```

Comment:
Can we still parse `--config key1=value,with,commas,key2=another,value,with,commas` after this change? I don't think we can.